Support scss absolute path resolution for url()

atlanteh opened the pull request:
Adding resolve-url-loader broke all apps using scss with centralized assets folder and all ```url(./assets/*.png)``` broke (#7023)  
This change allows apps to use ```url(/assets/*.png)``` and it would map to ```src/assets/*.png```

I tested this on my app which has an images directory and after upgrading to CRA 3.2.0 (from 2.1.2) it failed. After reading resolve-url-loader I added this option and changed the path to absolute ```(/assets/*.png)``` and everything started working again

ulrichb commented:
... build error (_Cannot find module '@babel/helper-create-regexp-features-plugin'_) seems to be unrelated with the change in this PR.

atlanteh commented:
Yes, it's not related. Build is broken for the last 24 hours

atlanteh commented:
@skrzepij thanks for approving the PR!
When can we expect this to land?

atlanteh commented:
any update on this? We can't run CI ever since this issue occurred as we need this fix in order for CI to work properly

mrmckeb commented:
@atlanteh Can you please rebase and push up again - the CI should work now.

atlanteh commented:
@mrmckeb All passed :)

mrmckeb commented:
I'm OK with this change, @ianschmitz or @iansu - any thoughts?

atlanteh commented:
@mrmckeb @ianschmitz @iansu 
If there are no objection, can we proceed with this PR?

atlanteh commented:
Any update?

atlanteh commented:
??

atlanteh commented:
Can this PR be merged please?

atlanteh commented:
@skrzepij @mrmckeb @ianschmitz @iansu 
Can you please reply? Why is there a delay in this? This PR was created over two months ago. It's a basic 3 words fixup.
Please let's get this merged

mrmckeb commented:
I've tagged this for 3.4 so someone should review and merge it before that release. Sorry for the delay, a lot of the team are only returning after Christmas break.

atlanteh commented:
Thanks!

atlanteh commented:
Can you guys make sure this gets into next release? We really need this fix, and we've been working with this fix locally for 3 months now and it works great

atlanteh commented:
Why is this moved to milestone 3.6? This is a very small change. This PR is waiting for almost 6 months now. Please merge it 🙏

arkn98 commented:
+1 Please get this merged.

arwagner commented:
Need this fix merged ASAP, please.

atlanteh commented:
@mrmckeb I added a test, though I see master is currently red, so my tests will fail as well.
I pretty much copy-pasted the bootstrap test and changed what is required. Both these tests cannot run locally due to node-sass is not installed exception, but I see that they pass in the behavior pipelines

atlanteh commented:
@mrmckeb This is the logo from the templates.. What do you mean by "It isn't loading"? Are you running it locally and you can see the results? If so, how?

mrmckeb commented:
Sorry @atlanteh, I don't know what I did - it's fine, you're right. Hope you're having a great weekend, I'll get this merged in very shortly.
